Makefile

```diff
@@ -13,6 +13,11 @@ MAKEFLAGS+=--no-builtin-rules
 # Project settings
 
 INSTANCE_PORT=8080
+export RAZZLE_DEV_PROXY_API_PATH=http://localhost:8080/Plone
```

Comment:
I don't know if hardcoding this here could have any implications, otherwise I can't see any now...

Reply:
It should reproduce the default.  It can also be overridden with `$ make RAZZLE_DEV_PROXY_API_PATH=... start` or by exporting a new `RAZZLE_DEV_PROXY_API_PATH` value in the environment and running `$ make -e start`.  Regardless, it's not necessary so I've commented them out.